index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <!-- custom style link -->
+<link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+     <!-- header start -->
+  <header class="navbar-container">
+        <h1>John Doe</h1>
+      <nav>
+        <div id="toggle">
+          <a href=""><img src="media/icons/Enabled.png" alt=""></a>
+        </div>
+        <ul>
+          <li><a href="index.html">Portfolio</a> </li>
+          <li><a href="about.html">About</a> </li>
+          <li><a href="contact.html">Contact</a> </li>
+          <li><a href="#"><img src="media/icons/Mail.png" alt=""></a></li>
+        </ul>
+      </nav>
+    
+  </header>
+     <!-- header end -->
+        <main>
+
+      <!-- headline section start -->
+<section class="headline-container">
+
+  <div class="headline-text">
+
+      <h1>Hey There.</h1>
+      <h1>I’m Alex</h1>
+      <h3>I’m a Software Developer</h3>
+      <p>I can help you build a product , feature or website. Look through some of my work  and experience! If you like what you see and have a project you need coded, 
+          don’t hesitate to contact me.</p>
+
+  </div>
+
+  <div class="social-media-logo">
+
+    <ul>
+      <li><a href=""><img src="media/icons/github.png" alt="gitHub icon"></a></li>
+      <li><a href=""><img src="media/icons/Linkedin icon.svg" alt="linkedin icon"></a></li>
+      <li><a href=""><img src="media/icons/angel list.png" alt="angel icon"></a></li>
+      <li><a href=""><img src="media/icons/twitter.svg" alt="twitter icon"></a></li>
+      <li><a href=""><img src="media/icons/medium.png" alt="medium icon"></a></li>
+    </ul>
+
+  </div>
+
+
+  <div class="headline-image">
+
+    <img src="media/images/Illustration-header.png" alt="">
+
+  </div>
```

Comment:
- [ ] You have done a great job on this headline section, apart from this image which you added as an image but should be added as a background image.